toucan_connectors/postgres/postgresql_connector.py

```diff
@@ -161,3 +162,10 @@ def get_status(self) -> ConnectorStatus:
                 )
 
         return ConnectorStatus(status=True, details=self._get_details(3, True), error=None)
+
+    def describe(self, data_source):
+        connection = pgsql.connect(**self.get_connection_params(database=data_source.database))
+        with connection.cursor() as cursor:
+            cursor.execute(f"""select * from ({data_source.query.replace(';','')}) as q LIMIT 0;""")
```

Comment:
I think, for everything related to sql, let keep all in uppercase like on snowflake !
'SELECT', 'FROM', ....
Even for column name, that will be easy for the future collaboration with snowflake source code i think !

Reply:
thx.